src/utils/formatters.ts

```diff
@@ -2,6 +2,13 @@ import { snakeCase } from 'lodash'
 import { serializeServiceArea } from './serializers'
 import { capitalizeWords } from './string'
 
+/**
+ * Formats text and capitalize first letters.
+ */
+export function capitalizeFirstLetter(text: string) {
```

Comment:
Looks like this PR is still a WIP, but see https://lodash.com/docs/4.17.4#capitalize